.github/workflows/main.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         include:
 
           - name: linux-python2
-            os: ubuntu-16.04
```

Comment:
You shouldn't need to do this - this commit was merged to RB-10.1 and needs to be merged through to master too. Let me take a look at doing that and then you should be able to drop it...

Reply:
OK, I've merged that through so I think things should be OK now. If you're interested, the strategy we use for managing/syncronising release branches is documented briefly [here](https://github.com/GafferHQ/gaffer/wiki/Branches-and-Versioning).

Reply:
Great, I rebased and the Ubuntu update commit is out. I'm still doing the PR to `master`, should I target it to 10.2 instead? When I just changed the target to `master` it included a bunch of other commits. Rebasing onto 10.2 seems to potentially open up a can of worms?

Reply:
It would be good to target `RB-10.2` if it's not too much hassle - it means we can drop Azure more quickly. I would expect `git rebase -i RB-10.2` to bring along a bunch of commits from master by default. You'd need to drop those from the list so it was just your commits, then force push your branch and then change the PR target to `RB-10.2`.

Reply:
Thanks for the guidance, looks like it's retargeted smoothly now.